source/release-notes.html.md

```diff
@@ -9,40 +9,42 @@ weight: 200
 
 Changes to the data:
 
-- Change the structure of an application:
+- Change the structure of an [application](/reference#get-applications):
   - add `type` field
   - add `attributes` field and move `status`, `submitted_at`, `updated_at`, `personal_statement`
     `candidate`, `contact_details`, `course`, `qualifications`, `work_experiences`
     `references`, `offer`, `withdrawal` and `rejection` fields into it
-- Remove date from reject endpoint
-- Rename the `org` attribute to `organisation_name` for the Work Experience resource
-- Limit candidates to only 2 nationalities
-- Rename the `type` field on `qualification` to `qualification_type`
-- Rename the `type` field on `reference` to `reference_type`
-- Add HESA ITT data to the application. Available only once a student is enrolled
+- Remove date from [reject](/reference/#post-applications-application-id-reject) endpoint
+- Limit [candidates](/reference/#candidate) to only 2 nationalities
+- Rename the `org` field on [work experience](/reference/#workexperience) to `organisation_name`
+- Rename the `type` field on [qualification](/reference/#qualification) to `qualification_type`
+- Rename the `type` field on [reference](/reference/#reference) to `reference_type`
+- Add HESA ITT data to the [application](/reference#get-applications). Available only once a student is enrolled
 
 Changes to functionality:
 
 - Change the successful response for all endpoints to be within a `data` object
-- Change the successful response for making an offer, confirming candidate enrolment,
-  confirming offer conditions are met and rejecting an application endpoints to be
-  the application
-- Change the HTTP response code for making an offer, confirming candidate enrolment,
-  confirming offer conditions are met and rejecting an application endpoints to `200`
-- Support an `order` parameter when [retrieving many applications](/retrieve-many-applications)
-- Support a `provider_code` parameter when [retrieving many applications](/retrieve-many-applications)
+- Change the successful response for [making an offer](/reference/#post-applications-application-id-offer),
+  [confirming candidate enrolment](/reference/#post-applications-application-id-confirm-enrolment),
+  [confirming offer conditions are met](/reference/#post-applications-application-id-confirm-conditions-met)
+  and [rejecting an application](/reference/#post-applications-application-id-reject) endpoints to:
+  - return the application
+  - return a HTTP `200` response code
+- Support an `order` and a `provider_code` parameter when [retrieving many applications](/reference#get-applications)
```

Comment:
You may want to improve the reliability of this, maybe something like this:

"Add support for ordering by the params `order` and a `provider_code` ..." 

Up to you anyway :)